src/System.Security.Cryptography.Encoding/src/System/Security/Cryptography/Base64Transforms.cs

```diff
@@ -120,13 +120,13 @@ public FromBase64Transform(FromBase64TransformMode whitespaces)
             _whitespaces = whitespaces;
         }
 
-        // Converting from Base64 generates 3 bytes output from each 4 bytes input block
-        private const int Base64InputBlockSize = 4;
         // A buffer with size 32 is stack allocated, to cover common cases and benefit from JIT's optimizations.
         private const int StackAllocSize = 32;
-        public int InputBlockSize => 1;
+
+        // Converting from Base64 generates 3 bytes output from each 4 bytes input block
+        public int InputBlockSize => 4;
         public int OutputBlockSize => 3;
-        public bool CanTransformMultipleBlocks => false;
+        public bool CanTransformMultipleBlocks => true;
```

Comment:
I'd like to see at least some test have to change as a result of this.  Can you add a test that just checks that the block sizes and can-transform match what we want them to be? :)

Reply:
Tests are added with 886b05b77245f5f738e10a64eb118183855b8365